src/pages/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Compression App</title>
+</head>
+<body>
+  <h1>Expense info</h1>
+
+  <form method="POST" action="/data" enctype="multipart/form-data">
+    <label>
+      Date:
+
+      <input type="date" name="date" value="2023-06-07" required>
+    </label>
+
+    <br></br>
```

Comment:
Avoid using <br> tags for styling, instead use CSS for presentation. Remove the <br> tags and use CSS for layout.